src/darwin/Framework/CHIP/templates/MTRCommandPayloadsObjc-src.zapt

```diff
@@ -225,6 +229,21 @@ NS_ASSUME_NONNULL_BEGIN
 @implementation MTR{{asUpperCamelCase parent.name preserveAcronyms=true}}Cluster{{asUpperCamelCase name preserveAcronyms=true}}Params (InternalMethods)
 
 {{#if (isStrEqual source "server")}}
+- (nullable instancetype)initWithDecodableStruct:(const chip::app::Clusters::{{asUpperCamelCase parent.name}}::Commands::{{asUpperCamelCase name}}::DecodableType &)decodableStruct
```

Comment:
Do we need to code-get this into every concrete class separately? Maybe this can just live in the base class